### PR TITLE
JDK-8267709: Investigate differences between HtmlStyle and stylesheet.css

### PR DESCRIPTION
**Pull request by jonathan-gibbons**
Please review a new test to track the differences between `HtmlStyle.java` and `stylesheet.css`.

The test passes, because all discrepancies are noted within the test. However, some of the discrepancies may indicate the need for follow-up work, such as adding new items to `HtmlStyle` for documentation purposes, or removing unused code in the main `stylesheet.css` file.

There is also a minor/trivial change to fix a copy-paste error in a comment in `HtmlStyle`.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8267709](https://bugs.openjdk.java.net/browse/JDK-8267709): Investigate differences between HtmlStyle and stylesheet.css

### Reviewers
 * [Hannes Wallnöfer](https://openjdk.java.net/census#hannesw) (@hns - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/4195/head:pull/4195` \
`$ git checkout pull/4195`

Update a local copy of the PR: \
`$ git checkout pull/4195` \
`$ git pull https://git.openjdk.java.net/jdk pull/4195/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 4195`

View PR using the GUI difftool: \
`$ git pr show -t 4195`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/4195.diff">https://git.openjdk.java.net/jdk/pull/4195.diff</a>

</details>

### CODE REVIEW

File: test/langtools/jdk/javadoc/doclet/checkStylesheetClasses/CheckStylesheetClasses.java [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+        styleSheetNames.remove("module-graph");
+
+        // apparently unused
+        // "tab" is commented implying it is in the header/footer, but
+        // (a) it is a poorly chosen name
+        // (b) it does not seem to be used in make/Docs.gmk or anywhere else
+        removeAll(styleSheetNames, "all-classes-container", "all-packages-container",
+                "bottom-nav", "clear", "constant-values-container", "deprecated-content",
+                "footer", "hidden", "override-specify-label", "serialized-class-details",
+                "tab", "table-sub-heading-color");
```
Comment (hns) [inline test/langtools/jdk/javadoc/doclet/checkStylesheetClasses/CheckStylesheetClasses.java:153]: Shouldn't these be removed? Are we planning to do that in a separate issue?

**Review by hns — APPROVED** [review]
The new test looks good, apart from the question whether we should remove unused styles now, later, or not at all.

**Comment by jonathan-gibbons** [comment]
> The new test looks good, apart from the question whether we should remove unused styles now, later, or not at all.

I was thinking to remove the unused styles in a separate PR. I wasn't sure if this would need extra investigation/verification that these styles are really unused.

**Pull request closed without merging**